tsconfig.base.json

```diff
@@ -53,6 +53,7 @@
       "@fluentui/react-tabster": ["packages/react-tabster/src/index.ts"],
       "@fluentui/react-text": ["packages/react-text/src/index.ts"],
       "@fluentui/react-theme": ["packages/react-theme/src/index.ts"],
+      "@fluentui/react-theme-sass": ["packages/react-theme-sass/src/index.ts"],
```

Comment:
There's no ts here, do we need path aliases ?

Reply:
good question, for sake of consistency I'd say yes. Also I'd keep the index.ts there with empty export / even better to throw a runtime error if someone would import that